finetuner-workflow/finetune-workflow.yaml

```diff
@@ -217,8 +226,7 @@ spec:
             - name: dest
               value: "/{{workflow.parameters.pvc}}/models/{{workflow.parameters.model}}"
             - name: tokenizer_only
-              value: "{{steps.check-model.outputs.result}}"
```

Comment:
Marking as resolved since the `or` has since been replaced: https://github.com/coreweave/kubernetes-cloud/pull/376/commits/d795de4caffb0e6af3f3b925aa4643702996a092